packages/e2e/test/web-extension/webpack.config.base.js

```diff
@@ -53,6 +53,10 @@ module.exports = {
         /@dcspark\/cardano-multiplatform-lib-nodejs/,
         '@dcspark/cardano-multiplatform-lib-browser'
       ),
+      new NormalModuleReplacementPlugin(
```

Comment:
Since we don't want to run blockfrost in extension, please try to
- uninstall `@emurgo/cardano-serialization-lib-asmjs`
- replace my changes in  `webpack.config.base.js` with [webpack externals](https://webpack.js.org/configuration/externals/#externals) configuration for `@cardano-sdk/blockfrost-client`, so that it's not bundled at all
```suggestion
```

Reply:
done in ed8c12c913d